doc/configuration.rst

```diff
@@ -859,6 +859,14 @@ document's title. In some cases, a user may wish to not remove titles when
 custom prefixes or other custom modifications are in play. By default, this
 option is enabled with a value of ``True``.
 
+confluence_single_page
+~~~~~~~~~~~~~~~~~~~~~~
+
+A boolean value to whether or not generate a single confluence page. Normally,
+each .rst file becomes its own confluence page. With this option enabled, every
+.rst file is consolidated into a single page. By default, this option is enabled
+with a value of ``False``.
+
```

Comment:
Some comments about this section:

1. The section `confluence_single_page` was added in the middle of the content for `confluence_remove_title` -- now serving the example for `confluence_remove_title` in this new section. The other configuration's example should be above and a new configuration example for `confluence_single_page` should be added. 
2. Minor notes: this section hints of `.rst` files, but that extension is option for a Sphinx set; the word `confluence` should be capitalized in the paragraph; and, the default state in the text claims the enablement of this feature is when the value is `False`. If I may suggest the following instead:

   > A boolean value to whether or not generate a single Confluence document. Normally, each source document (e.g. \`\`.rst\`\`) will be translated into their own respective Confluence-supported document. With this option enabled, every source document is consolidated into a single Confluence-supported document. By default, this option is disabled with a value of \`\`False\`\`.

   Does that work for you?

Reply:
works for me. done.